pkg/model/model.go

```diff
@@ -296,8 +344,61 @@ func (m *Model) GetCRDs() ([]*CRD, error) {
 		// Now add the additional printer columns that have been defined explicitly
 		// in additional_columns
 		crd.addAdditionalPrinterColumns(m.cfg.GetAdditionalColumns(crdName))
-
 		crds = append(crds, crd)
+
+		for _, toInjectNestedFields := range alltoInjectNestedFields {
+
+			addThis := ""
```

Comment:
need a better name than `addThis`

Reply:
Changed the name to `newCustomField`. Let me know if that sounds okay.